src/Layout/with-layout.spec.js

```diff
@@ -0,0 +1,39 @@
+// @flow weak
+/* eslint-env mocha */
+/* eslint react/prop-types: off */
+
+import React from 'react';
+import { assert } from 'chai';
+import { createMountWithContext } from 'test/utils';
+import withLayout, { styleSheet } from './with-layout';
+
+describe('withFlex HOC', () => {
```

Comment:
naming. I am not 100% set on choosing `withLayout` over `withFlex` for the HOC. Care to suggest a name?